README.md

```diff
@@ -1,37 +1,112 @@
-causal genes, thus hampering the identification of good targets for drug
-development or repurposing.
+## Overview
 
-PhenoPLIER is a computational approach that integrates TWAS and pharmacological
-perturbations with gene co-expression patterns (gene modules) learned from large
-expression compendia across different tissues, cell types and other conditions.
-PhenoPLIER aims to find novel gene module associations with complex traits, and
-also new pharmarcological perturbations that can suggest alternative treatment
-options when some genes are not druggable. The approach allows to infer 1) how
-gene modules affect complex traits, 2) in which conditions (cell types, tissues,
-etc) are those genes activated, and 3) how a gene module activity could be
-modulated with different compounds.
+![](images/phenoplier_overview.png)
 
-# Setup
+PhenoPLIER is new computational strategy that integrates statistical associations from GWAS/TWAS with groups of genes (gene modules) that have similar expression patterns across the same cell types.
+This allows us to go beyond gene-trait statistical associations and infer the cell types where gene expression regulation is likely to be disrupted, resulting in cell type-specific pathology.
 
-To prepare the environment to run the PhenoPLIER code, follow the steps in [environment](environment/).
-That will create a conda environment and download the necessary data.
+For more details, check out our manuscript in [bioRxiv](https://doi.org/10.1101/2021.07.05.450786) or our [Manubot web version](https://greenelab.github.io/phenoplier_manuscript/).
 
-# Running code
+## Data
 
-Once the environment is ready, you can start your JupyterLab server by running:
+You can access all the data generated by this project by going to the [`data/`](data/)
+folder. Please, carefully follow instructions for citations, since this projects
+uses data generated by others.
+
+## Setup
+
+To prepare the environment to run the PhenoPLIER code, follow the steps in
+[environment](environment/). That will create a conda environment and download
+the necessary data. Alternatively, you can use our Docker image (see below).
+
+## Running code
+
+### From command-line
+
+First of all, export your settings to environmental variables, so non-Python scripts
+can access them:
+```bash
+eval `python libs/conf.py`
+```
+
+The code to preprocess data and generate results is in the `nbs/` folder. All
+notebooks are organized by directories, such as `01_preprocessing`, with file
+names that indicate the order in which they should be run (if they share the prefix, then it
+means they can be run in parallel). For example, to run
+all notebooks for the preprocessing step, you can use this command (requires
+[GNU Parallel](https://www.gnu.org/software/parallel/)):
 
 ```bash
-bash scripts/run_nbs.sh
+cd nbs/
+parallel -k --lb --halt 2 -j1 'bash run_nbs.sh {}' ::: 01_preprocessing/*.ipynb
```

Comment:
This is very weird and it might be the reason why you get errors with pytest. My guess is that something went wrong with the conda environment installation. The `environment.yml` file specifies `conda-forge` as the channel, I wonder if your conda is somehow overriding that?

Reply:
Yeah that's probably it, since I'm not installing conda from scratch my conda configuration is probably different from the default